### PR TITLE
THOR-943

### PR DESCRIPTION
**Pull request by rartiss55**
- Combining filter tray and filter builder.
- Removing filter builder as a visualization.

### CODE REVIEW

**Comment by wagnerwj** [comment]
Nag Thomas about this one, Rachel. I'd like to see this sooner.

**Review by Konnoke — COMMENTED** [review]
What are the advantages of having the filter component handle the filter builder and current filter components vs having the app component handle it?

File: src/app/components/filters/filters.component.ts
Comment (Konnoke) [inline src/app/components/filters/filters.component.ts]: I don't see lodash being used in the filter component.

**Comment by rartiss55** [comment]
@Konnoke I thought it was better for code maintenance and testability to have a separate component
manage the filters rather than adding it to the app component. Plus it would make it easier if both filter components were further combined later.

**Review by ThomasSchellenbergNextCentury — DISMISSED** [review]

**Comment by Konnoke** [comment]
Expected behavior, but old save states that have the filter builder component in the dashboard will still load but the filter builder component in the dashboard will be blank (visualization container component with nothing in it)

**Comment by rartiss55** [comment]
@Konnoke I think that's expected because we're phasing out the filter builder as a visualization. @ThomasSchellenbergNextCentury does this sound fine to you?

**Review by Konnoke — DISMISSED** [review]

**Review by Konnoke — DISMISSED** [review]

File: src/app/components/filters/filters.component.ts
Comment (rartiss55) [reply src/app/components/filters/filters.component.ts]: you're right -- removed.

**Comment by ThomasSchellenbergNextCentury** [comment]
@rartiss55 @Konnoke 

I think a saved state loading a blank widget in place of the filter builder is fine.  If it becomes an issue later then we can probably implement something to detect it and ignore it.

**Review by ThomasSchellenbergNextCentury — DISMISSED** [review]

**Comment by rartiss55** [comment]
@ThomasSchellenbergNextCentury merged in lorelei-master but the way I did it must have cancelled out approvals. Please re-approve when you get the chance. Thanks!

**Review by ThomasSchellenbergNextCentury — APPROVED** [review]

**Pull request merged**
